label_studio_sdk/project.py

```diff
@@ -1770,3 +1773,36 @@ def export_snapshot_download(self,
                 for chunk in response:
                     f.write(chunk)
         return response.status_code, filename
+
+    def get_files_from_tasks(self,
+                             tasks: Dict,
+                             get_tasks: bool = False
+                             ):
+        """ Copy files from tasks to cache folder
+
+        Parameters
+        ----------
+        tasks: Dict
+        Tasks to download to local storage
+        get_tasks: bool
+        Get all tasks from current project
+
+        Returns
+        -------
+        list
+            List of filenames
+        """
+        if get_tasks:
+            tasks = self.get_tasks()
+        filenames = []
+        if tasks:
+            for task in tasks:
+                for key in task['data']:
+                    try:
+                        filename = get_local_path(task['data'][key],
```

Comment:
I'm not sure we need to fetch all data keys - not all of them are file paths. Can we check first whether it's filepath-like string?

Reply:
It's not only filepath - it can be S3 or other URL, all the more we can modify get_local_path to work with object of other types so this check will cut off these types.